gammapy/datasets/io.py

```diff
@@ -53,15 +53,15 @@ class OGIPDatasetWriter(DatasetWriter):
 
     The naming scheme is fixed as following:
 
-    * PHA file is named filename.fits
-    * BKG file is named filename_bkg.fits
-    * ARF file is named filename_arf.fits
-    * RMF file is named filename_rmf.fits
+    * PHA file is named filename.fits.
+    * BKG file is named filename_bkg.fits.
+    * ARF file is named filename_arf.fits.
+    * RMF file is named filename_rmf.fits.
```

Comment:
I think here we should not have a full stop at the end. It looks odd when it is a filename

Reply:
Agree, it did kinda look bad...